tutorial/sample-problems/ProblemTechniques/WeightedGrader.pg

```diff
@@ -11,33 +11,38 @@
 ## MO(1)
 ## KEYWORDS('weighted grader')
 
-# created as a full problem by Peter Staab  2023.06.02
+# created as a full problem by Peter Staab 2023.06.02
 
 #:% name = Weighted Grader
 #:% type = [technique]
 #:% categories = [grader]
 
 #:% section = preamble
+#: Load the PODLINK('weightedGrader.pl') macro.
 DOCUMENT();
 loadMacros('PGstandard.pl', 'PGML.pl', 'weightedGrader.pl', 'PGcourse.pl');
 
-#:% section=setup
-#: Call `install_weighted_grader();` so that the weighted grader is used.
+#:% section = setup
+#: Call `install_weighted_grader` so that the weighted grader is used.
```

Comment:
Since the weighted grader is not in the avg_grader code, should we update this (and maybe other problems) to at least remove the need for the macro?

Reply:
I updated the problems that were using the `weightedGrader.pl` macro to instead use the default `avg_problem_grader` with weights.

Reply:
Oops.  I missed this problem though.  I will fix that.

Reply:
Okay, now I think I found all instances that were using the `weightedGrader.pl` macro.